x/ccv/provider/keeper/keeper.go

```diff
@@ -178,6 +178,50 @@ func (k Keeper) DeleteChainToChannel(ctx sdk.Context, chainID string) {
 	store.Delete(types.ChainToChannelKey(chainID))
 }
 
+// SetProposedConsumerChain sets consumer chainId in gov consumerAddition proposal in store
+// the consumer chainId is only added when the voting period for consumerAddition proposal
```

Comment:
Is this what you meant?
```suggestion
// This consumer chainId is deleted once the voting period for the proposal ends
```